.gitignore

```diff
@@ -1,33 +1,8 @@
-### PyDev ###
-*.pydevproject
-.project
-.metadata
-bin/**
-tmp/**
-tmp/**/*
-*.tmp
-*.bak
-*.swp
-*~.nib
-local.properties
-.classpath
-.settings/
-.loadpath
-
-# CDT-specific
-.cproject
+!.pylintrc
```

Comment:
Had to whitelist because apparently there was another rule conflicting with letting me add this file to VC.